src/index.html

```diff
@@ -13,6 +33,44 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article>
+      <div
+        data-qa="card"
+        class="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Product image"
+        />
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+          <p class="card__code">Product code: 195434</p>
+        </h2>
+
+        <div class="stars stars--4">
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <p class="stars-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          Price:
+          <div class="card__price-value">$2,199</div>
+        </div>
```

Comment:
You need to use semantic tag for text
```suggestion
        <div class="card__price">
          <p class="card__price-label">Price:</p>
          <p class="card__price-value">$2,199</p>
        </div>
```